### PR TITLE
Segmented download page

### PR DESCRIPTION
**Pull request by ArneBab**
This moves the note below the download instructions: We first and foremost want people to install Freenet, not to think that it’s complicated.

### CODE REVIEW

File: pages/download.py [hunk 1]
```diff
@@ -188,13 +182,26 @@ def get_content(self):
...
+        self.title = _("Using Freenet")
+        self.slug = "usage"
+    def get_content(self):
+        # License for all content in this section: GFDL (from old freenetproject.org website)
+        return text(md(_("""
+[Step by step guide](http://freesocial.draketo.de/) to setting up Freenet and
+various Freenet apps. Please try this, especially if installing on OS X. We
+are not responsible for unofficial third party apps it recommends (including
+FMS), but many Freenet users and developers use them.
+""") + _("""
```
Comment (bertm) [inline pages/download.py]: You're missing a `"\n\n"` here. `### HOWTO` ends up as text in the previous paragraph without this.

File: pages/download.py [hunk 2]
```diff
@@ -188,13 +182,26 @@ def get_content(self):
...
 ### HOWTO
```
Comment (bertm) [inline pages/download.py]: While we're at it, there's no such word as "HOWTO". It's spelled "how-to", but even that doesn't remotely seem to capture the subject of this paragraph ;-)

**Comment by bertm** [comment]
I figured I could easily fix that myself while I was at it. Does this look alright to you, @ArneBab?

**Comment by ArneBab** [comment]
I think it looks good, yes. It sounds better than the bullet-style sentence which was there before.

**Comment by bertm** [comment]
Why was the rabbit graphic merged as part of this pull request? It seems unrelated.

**Comment by ArneBab** [comment]
because I failed at separating those.

**Pull request merged**